### PR TITLE
Update dotnet.md

### PR DESCRIPTION
**Pull request by eevahr**
Add instructions for updating the dotnet-ef tools

### CODE REVIEW

**Comment by ajcvickers** [comment]
Thanks @eevahr!

File: entity-framework/core/miscellaneous/cli/dotnet.md [hunk 1]
```diff
@@ -121,6 +121,12 @@ Entity Framework Core .NET Command-line Tools 2.1.3-rtm-32065
 <Usage documentation follows, not shown.>
 ```
 
+
+## Updating the tools
+
+Use `dotnet tool update --global dotnet-ef` to update the tools to the latest available version, install a specific version by appending `--version <VERSION>` to your command.
```
Comment (smitpatel) [inline entity-framework/core/miscellaneous/cli/dotnet.md:127]: It should differentiate between updating global tool vs local tool.

Comment (eevahr) [reply entity-framework/core/miscellaneous/cli/dotnet.md:127]: What is a local tool and how do you update it?

Comment (eevahr) [reply entity-framework/core/miscellaneous/cli/dotnet.md:127]: Is it referring to running `dotnet tool update dotnet-ef` without the --global parameter?

Comment (smitpatel) [reply entity-framework/core/miscellaneous/cli/dotnet.md:127]: https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools

**Pull request merged**